docs/getting-started.md

```diff
@@ -38,6 +38,8 @@ Lastly, install the React Native for Windows packages.
 npx react-native-windows-init --overwrite
 ```
 
+> The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect.

```

Comment:
"temporary issue" should link to an issue tracking this being different.

Reply:
do we have an issue on the main repo tracking that or should I go ahead and create one?

Reply:
I don't think we have one. Go ahead and create one and link to it here.

Reply:
Now that the site is versioned, you should add this text to the getting-started guides under 60 and 61.

Reply:
created it here: https://github.com/microsoft/react-native-windows/issues/4698